loader/vk_loader_platform.h

```diff
@@ -433,12 +433,20 @@ static inline char *loader_strncpy(char *dest, size_t dest_sz, const char *src,
 static inline const char *LoaderPnpDriverRegistry() {
     BOOL is_wow;
     IsWow64Process(GetCurrentProcess(), &is_wow);
+#ifdef VULKANSC
+    return is_wow ? "VulkanSCDriverNameWow" : "VulkanSCDriverName";
+#else
     return is_wow ? "VulkanDriverNameWow" : "VulkanDriverName";
+#endif
 }
 static inline const wchar_t *LoaderPnpDriverRegistryWide() {
     BOOL is_wow;
     IsWow64Process(GetCurrentProcess(), &is_wow);
+#ifdef VULKANSC
+    return is_wow ? L"VulkanSCDriverNameWow" : L"VulkanSCDriverName";
+#else
     return is_wow ? L"VulkanDriverNameWow" : L"VulkanDriverName";
+#endif
 }
```

Comment:
Alternatively we could modify the registry keys on lines 194-197 to put VulkanSC in the path instead of the actual key name (it might be less intrusive overall - as there are currently some tests that are failing with this change). 

And similarly for the keys below, otherwise.

Reply:
Yes, that probably should be preferred.

Reply:
Looking at https://github.com/KhronosGroup/VulkanSC-Loader/blob/sc_main/docs/LoaderLayerInterface.md#windows-layer-discovery it looks like both sets of keys are needed. 

I've suggested that in the case where we modify the PNP registry keys.. it would be preferable to modify how the Key is selected, so you don't need to ifdef each of the 6 different cases

eg if you could do something like "Vulkan" VARIANT "DriverNameWow"
and then VARIANT was defined as "" in the Vulkan case, and "sc" in the VulkanSC case

Reply:
Made these changes in the most recent commit. Thanks for reviewing!

Reply:
> eg if you could do something like "Vulkan" VARIANT "DriverNameWow"

Makes sense, but only if this change will actually get upstreamed. But the name `API_VARIANT` seems to generic and would prefer at least a `VK` prefix or something to avoid potential name collisions.

Also, defining the `_W` version for the widechar support should really be limited to Windows platforms.

Reply:
We should also update the LoaderLayerInterface.md document to specify the correct keys for VKSC.